README.md

```diff
@@ -204,6 +205,10 @@ To run the tests against a remote deployment you will need to specify the enviro
 
 `EQ_FUNCTIONAL_TEST_ENV=https://staging-new-surveys.dev.eq.ons.digital/ yarn test_functional`
 
+To run the census functional tests within the cypress UI:
+
+./node_modules/cypress/bin/cypress open
```

Comment:
I think it would be nicer to add it to yarn and add a yarn command here to keep it consistent the rest of the app.